backend/routes.js

```diff
@@ -424,6 +424,16 @@ export default class Servers {
             });
         });
 
+        server.post('/connections/:connectionId/sql-schemas', function schemasHandler(req, res, next) {
+            Datastores.schemas(
+                getConnectionById(req.params.connectionId)
+            ).then(schemas => {
+                res.json(200, schemas);
+            }).catch(error => {
+                res.json(500, {error: {message: error.message}});
+            });
+        });
+
```

Comment:
can we add a simple test for this endpoint? Here's a similar test: https://github.com/plotly/plotly-database-connector/blob/master/test/backend/routes.spec.js#L419